qcp/gates.py

```diff
@@ -95,7 +94,6 @@ def control_x(size: int, controls: List[int], target: int) -> Matrix:
     """
     Constructs a (2**size by 2**size) control-x gate with
     given controls and target
-
```

Comment:
These lines actually need to be here for Sphinx to convert the docstrings into HTML properly